File: Projeto_AlunoProf_BD/src/modelo/Professor.java

package modelo;

public class Professor extends Pessoa{
    private int siape;

    public Professor(String nome, String sexo, String cpf, int idade){
        super.setNome(nome);
        super.setSexo(sexo);
        super.setCpf(cpf);
        super.setIdade(idade);
    }

    public Professor() {}
    
    public int getSiape() {
        return siape;
    }
    public void setSiape(int siape) {
        this.siape = siape;
    }
}

File: Trabalho_Web2_AlunoProf/modelo/Professor.php
<?php
	require_once 'Pessoa.php';
	
	class Professor extends Pessoa{
		private $siape;
		
		public function setSiape($si){
			$this->siape = $si;
		}
		
		public function getSiape(){
			return $this->siape;
		}
	}
?>